README.md

```diff
@@ -1,77 +1,30 @@
-# 1Password Connect
+<p align="center">
+  <img src="https://github.com/1Password/connect/assets/46452606/0f7cf2a8-a290-41fc-b78d-3dfb1017f9be" alt="Header" >
+</p>
 
-The 1Password Connect server provides a REST API that can be used to securely access data from your 1Password account. Learn more about [setting up a Secrets Automation Workflow](https://support.1password.com/secrets-automation/)
+<p align="center">
+  <a href="https://1password.com">
+      <h3 align="center">1Password Connect</h3>
+  </a>
+</p>
 
-## Quick Start
+<p align="center">
+ <h5 align="center">Access your 1Password secrets using 1Password Connect</h5>
+</p>
 
-### Create Server and Access Token
+<p align="center">
+  <a href="https://developer.1password.com/docs/connect/get-started"><img alt="Get Started/Quick start" src="https://user-images.githubusercontent.com/45081667/226940040-16d3684b-60f4-4d95-adb2-5757a8f1bc15.png" height="37" /></a>
+</p>
+<br/>
 
-You can use the [1Password CLI](https://1password.com/downloads/command-line/) to set up a 1Password Connect server and issue tokens for it.
+---
 
-First, set up a 1Password Connect server:
+With 1Password Connect, you can securely access your 1Password items and vaults in your company's apps and cloud infrastructure using a private REST API.
+## Get started
+[Check out our developer docs to get started.](https://developer.1password.com/docs/connect/get-started)
+## 💙 Community & Support
 
-```sh
-op connect server create <name> --vaults <vault>[,<vault>]
-```
+- File an [issue](https://github.com/1Password/connect/issues) for bugs and feature requests
```

Comment:
Do we want to link to the issues list, or maybe to the form to create a new issue https://github.com/1Password/connect/issues/new?

Reply:
IMO linking to the issues list is preferable; hopefully it will encourage people to review/search the issues first to see if someone has already posted a similar issue before they go creating a new one.

Reply:
Yeah that makes the most sense to me as well